src/packages/sdk/src/utils/find.ts

```diff
@@ -0,0 +1,290 @@
+/* eslint-disable @typescript-eslint/no-inferrable-types */
```

Comment:
Oh so this is a duplicated file 🤔 

Possible to avoid duplication? 

Like having only one in sdk and importing it in client?

Reply:
Just copy pasted my comment above:

> Btw that file is duplicated and that is normal, it's related to another issue where we are not able to generate types for cases where we import from another local package. For example, it would fail if sdk would export find and find is imported in client. This only fails on functions and is related to rollup-plugin-dts. Will open an issue soon.